HW0/P6/P6B.py

```diff
@@ -16,18 +17,33 @@ def burnTime(t):
 
     # Use a variety of wait times
     ratio = []
-    wait_time = []
+    wait_time = np.logspace(-6,0,7);
+    print wait_time;
 
     for t in wait_time:
         # Compute jobs serially and in parallel
         # Use time.time() to compute the elapsed time for each
-        serialTime = 1
-        parallelTime = 1
+        # Compute serial time
+        start = time.time();
+        for i in range(0,16):
```

Comment:
I would use N instead of 16 here -- we've already defined it, so no need to have random "magic" constants.